DiceKitTests/MultiplicationExpression_Tests.swift

```diff
@@ -113,9 +113,67 @@ extension MultiplicationExpression_Tests {
         expect(result.multiplierResult) == expectedMultiplierResult
         expect(result.multiplicandResults) == expectedMultiplicandResults
     }
+
+}
+
+// MARK: - Operators
+extension MultiplicationExpression_Tests {
+    
+    /// Same as test_evaluate_shouldCreateResultCorrectly(),
+    /// but using an operator overload instead of MultiplicationExpression
+    func test_operator_shouldCreateMultiplicationExpression() {
+        let multiplicandResults = [8, 4, 2, -7, 1, 9, 3, 3, -3, 4, 5]
+        let expectedMultiplicandResults = multiplicandResults
+        let leftExpression = expectedMultiplicandResults.count
+        let expectedMultiplierResult = leftExpression
+        let mockRightExpression = MockExpression()
+        mockRightExpression.stubResulter = { multiplicandResults[mockRightExpression.evaluateCalled] }
+        let expression = leftExpression * mockRightExpression
+        
+        let result = expression.evaluate()
+        
+        expect(result.multiplierResult) == expectedMultiplierResult
+        expect(result.multiplicandResults) == expectedMultiplicandResults
+    }
+    
+    func test_operator_shouldWorkWithIntAndDie() {
+        let constant = 2
+        let die = d()
+        
+        let expression = constant * die
+        
+        expect(expression.multiplier) == constant
+        expect(expression.multiplicand) == die
+    }
+    
+    func test_operator_shouldNotBreakIntAndInt() {
+        let notAnExpression = 6 * 7
+        
+        expect(notAnExpression) == 42
+    }
+    
+    func test_operator_shouldWorkWithDieAndDie() {
+        let d4 = d(4)
+        let d6 = d()
+        
+        let expression = d4 * d6
+        
+        expect(expression.multiplier) == d4
+        expect(expression.multiplicand) == d6
+    }
+    
+    func test_operator_shouldWorkWithDieAndInt() {
```

Comment:
I would combine this with `test_operator_shouldWorkWithIntAndDie` when you make it a SwiftCheck, just like my comment above.